src/RollbarLogHandler.php

```diff
@@ -73,55 +74,11 @@ public function log($level, $message, array $context = [])
             return;
         }
 
-        $context = $this->addContext($context);
```

Comment:
Removing `addContext` removes much of the usefulness of this package for providing Laravel-specific debug information. 

What's your reasoning behind removing this valuable part of this package?